### PR TITLE
Added additional amp-analytics vars.

### LINKED ISSUE #1503: Support for additional substitution variables in amp-analytics
suggested by @msukmanowsky:
- [ ] availScreenWidth = screen.availWidth
- [ ] availScreenHeight = screen.availHeight
- [ ] colorDepth = screen.colorDepth

suggested by @avimehta:
- [ ] documentEncoding
- [ ] userLanguage

### PR DESCRIPTION
**Pull request by avimehta**
New vars: available screen width, available screen height, color depth,
document encoding and user language.

Fixes #1503

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -48,7 +48,12 @@ export const ANALYTICS_CONFIG = {
       'pageDownloadTime': 'PAGE_DOWNLOAD_TIME',
       'redirectTime': 'REDIRECT_TIME',
       'domInteractiveTime': 'DOM_INTERACTIVE_TIME',
-      'contentLoadTime': 'CONTENT_LOAD_TIME'
+      'contentLoadTime': 'CONTENT_LOAD_TIME',
+      'availableScreenHeight': 'AVAILABLE_SCREEN_HEIGHT',
+      'availableScreenWidth': 'AVAILABLE_SCREEN_WIDTH',
+      'screenColorDepth': 'SCREEN_COLOR_DEPTH',
+      'browserLanguage': 'BROWSER_LANGUAGE',
+      'documentCharset': 'DOCUMENT_CHARSET'
```
Comment (dvoytenko) [inline extensions/amp-analytics/0.1/vendors.js]: It's totally fine and even recommended to have a trailing comma. Please add.

File: src/url-replacements.js [hunk 2]
```diff
@@ -152,6 +152,33 @@ class UrlReplacements {
...
+    // Returns document characterset.
+    this.set_('DOCUMENT_CHARSET', () => {
+      const doc = this.win_.document;
+      return doc.characterSet || doc.charset;
+    });
+
+    // Returns the browser language.
+    this.set_('BROWSER_LANGUAGE', () => {
+      const nav = this.win_.navigator;
+      return (nav.language || nav.browserLanguage || '').toLowerCase();
```
Comment (dvoytenko) [inline src/url-replacements.js]: I believe a form `navigator.userLanguage` also exists.

**Comment by dvoytenko** [comment]
LGTM with one minor comment.

**Comment by avimehta** [comment]
ready for merge. Thanks!

File: extensions/amp-analytics/0.1/vendors.js (hunk 1 above)
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: didn't know that. done. Thanks!

File: src/url-replacements.js (hunk 2 above)
Comment (avimehta) [inline src/url-replacements.js]: added that as well.

**Pull request merged**